wallet/src/repository/keys.rs

```diff
@@ -12,10 +12,10 @@ pub fn wallet_name() -> &'static str {
     "name"
 }
 
-/// A wallet's caption.
+/// A wallet's description.
 #[inline]
-pub fn wallet_caption() -> &'static str {
-    "caption"
+pub fn wallet_description() -> &'static str {
+    "description"
```

Comment:
Any changes to the database keys are BREAKING CHANGE

Reply:
In this case it's not a breaking change because `wallet_description` was always read using `get_opt`, so nothing breaks if it does not exist, and the old `wallet_caption` was not used, so nothing breaks if we delete it.

Reply:
Yep